lib/pages/new_group/selected_contacts_map_change_notiifer.dart

```diff
@@ -1,16 +1,19 @@
 
+import 'package:fluffychat/pages/dialer/pip/dismiss_keyboard.dart';
 import 'package:fluffychat/presentation/model/presentation_contact.dart';
 import 'package:flutter/widgets.dart';
 
 class SelectedContactsMapChangeNotifier extends ChangeNotifier {
   final Map<PresentationContact, ValueNotifier<bool>> selectedContactsMap = {};
   final haveSelectedContactsNotifier = ValueNotifier(false);
 
-  Iterable<PresentationContact> get contactsList 
+  Iterable<PresentationContact> get contactsList
     => selectedContactsMap.keys
       .where((contact) => selectedContactsMap[contact]?.value ?? false);
 
-  void onContactTileTap({required PresentationContact contact}) {
+  void onContactTileTap({required PresentationContact contact, required BuildContext context}) {
```

Comment:
all are `required`, why need `{ }`? I will take time of the dev :))). a small point.